tests/5.1/target_teams_distribute_parallel_for/test_target_teams_distribute_parallel_for_default.c

```diff
@@ -0,0 +1,89 @@
+//-----------------test_target_teams_distribute_parallel_for_default.c-----
+//
+// OpenMP API Version 5.1 Mar 2023
+//
+// This test intends to test the functionality of an indirect
+// declare target region. A random int is chosen that correlates
+// to a function pointer. That function is then chosen and the
+// test ensures it runs as expected.
+// --------------------------------------------------------------
+
+
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+
+int DefaultFirstPrivate() {
+  int ErrCount = 0;
+  int Arr[32];
+  for (int i = 0; i < 32; ++i) {
+    Arr[i] = i;
+  }
+
+#pragma omp target teams distribute parallel for num_teams(2) thread_limit(10)\
+        default(firstprivate) map(tofrom: ErrCount)
+  for (int i = 0; i < 32; ++i) {
+    if (Arr[i] != i) {
+#pragma omp atomic
+      ErrCount += 1;
+    }
+  }
+  return ErrCount;
+}
+
+int DefaultPrivate() {
```

Comment:
This test does not actually check the `default(private)` behavior; one way to check the behavior is to check the value of `CONST` after the target region, which should not have been modified.

Reply:
Hi @seyonglee ,
Added check for CONST for its original value as suggested...
Thank you!!